projects/valtimo/dossier-management/src/lib/components/dossier-management-processes/dossier-management-processes.component.ts

```diff
@@ -79,47 +85,56 @@ export class DossierManagementProcessesComponent {
     },
   ];
 
+  private readonly _subscriptions = new Subscription();
+
   constructor(
     private readonly documentService: DocumentService,
     private readonly iconService: IconService,
+    private readonly dossierDetailService: DossierDetailService,
     private readonly route: ActivatedRoute,
     private readonly notificationService: NotificationService,
     private readonly translateService: TranslateService
   ) {
     this.iconService.register(Link16);
   }
+  public ngOnDestroy(): void {
+    this._subscriptions.unsubscribe();
+  }
 
   public deleteProcessDocumentDefinition(
     processDocumentDefinition: ProcessDocumentDefinition
   ): void {
-    this.documentService
-      .deleteProcessDocumentDefinition({
-        documentDefinitionName: processDocumentDefinition.id.documentDefinitionId.name,
-        processDefinitionKey: processDocumentDefinition.id.processDefinitionKey,
-        canInitializeDocument: processDocumentDefinition.canInitializeDocument,
-        startableByUser: processDocumentDefinition.startableByUser,
-      })
-      .subscribe({
-        next: () => {
-          this.notificationService.showNotification({
-            type: 'success',
-            title: this.translateService.instant(
-              'dossierManagement.processLinkNotification.unlinkSuccess'
-            ),
-            duration: 5000,
-          });
-          this.loadProcessDocumentDefinitions();
-        },
-        error: () => {
-          this.notificationService.showNotification({
-            type: 'error',
-            title: this.translateService.instant(
-              'dossierManagement.processLinkNotification.unlinkFailure'
-            ),
-            duration: 5000,
-          });
-        },
-      });
+    this._subscriptions.add(
+      this.documentService
+        .deleteProcessDocumentDefinition({
```

Comment:
Why add this to _subscriptions? deleteProcessDocumentDefinition is a http call that closes after 1 emission